.travis.yml

```diff
@@ -13,7 +13,7 @@ script:
     - npm run test
     - echo -e "\x1b\x5b35;1m*** Tests complete\x1b\x5b0m"
     - echo -e "\x1b\x5b35;1m*** Starting eslint...\x1b\x5b0m"
-    - npm run lint
+    - npm run lint -- -- --report-unused-disable-directives --max-warnings 0 .
     - cd ../../..
-    - npm run lint
+    - npm run lint -- --report-unused-disable-directives --max-warnings 0 .
```

Comment:
Is the last `.` needed?

Reply:
Yes. If you write a specific file in the location where dot is, lint is performed only on that file.
Dot seems to mean all files under the current directory recursively.

Reply:
I meant `npm run lint` already has `eslint .` so I thought this may be redundant.